### PR TITLE
Reset btn

### PR DESCRIPTION
**Pull request by vishak-kavalur**

### CODE REVIEW

**Comment by vishak-kavalur** [comment]
Merge is fine

**Review by amstel91 — COMMENTED** [review]
Can you update the plunker and share

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ $scope.otpInput={
             },
             onChange: function(value){
                 console.log(value);
-            }
+            },
+            reset: function(){}
```
Comment (amstel91) [inline README.md:37]: we are not allowing to pass a reset function. Example should be of how to reset it

**Comment by amstel91** [comment]
Needs work

**Pull request closed without merging**